### PR TITLE
Piranha

### PR DESCRIPTION
**Pull request by bgruening**
This work is a result from our Freiburger Dev-Day. Thanks to Pavan for working on this tool.

### CODE REVIEW

**Comment by bgruening** [comment]
@smithcr @TorHou review appreciated. Micha needs this one.

File: tools/piranha/piranha.xml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            <param name="merge" type="select" label="Merge significant bins within certain distance?">
+                <option value="yes">Yes</option>
+                <option value="0" selected="true">No</option>
+            </param>
+            <when value="yes">
+                <param name="cluster_dist" type="integer" value="1" label="Merge significant bins within this distance"
+                help="Default is 1 (merge adjacent)"/>
+            </when>
+        </conditional>
+        <param name="suppress_covars" type="boolean" checked="False" truevalue="-r" falsevalue="" label="Don't print covariate values in output" />
```
Comment (TorHou) [inline tools/piranha/piranha.xml]: I usually find the positive formulation easier to comprehend:
so: label="Print covariate values in output" checked="True" truevalue="" falsevalue="-r"

File: tools/piranha/piranha.xml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            <option value="NegativeBinomial">NegativeBinomial</option>
+            <option value="ZeroTruncatedPoisson">ZeroTruncatedPoisson</option>
+            <option value="ZeroTruncatedNegativeBinomial">ZeroTruncatedNegativeBinomial</option>
+            <option value="PoissonRegression">PoissonRegression</option>
+            <option value="NegativeBinomialRegression">NegativeBinomialRegression</option>
+            <option value="ZeroTruncatedPoissonRegression">ZeroTruncatedPoissonRegression</option>
+            <option value="ZeroTruncatedNegativeBinomialRegression">ZeroTruncatedNegativeBinomialRegression</option>
+        </param>
+        <param name="fitMethod" type="boolean" checked="False" truevalue="-t"
+            falsevalue="" label="Component fitting method" help=""/>
```
Comment (TorHou) [inline tools/piranha/piranha.xml]: maybe label="Use component fitting method." ?

File: tools/piranha/piranha.xml [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                <option value="yes">Yes</option>
+                <option value="no" selected="true">No</option>
+            </param>
+            <when value="yes">
+                <param name="model" type="data" multiple="false" format="xml" label="Select the model file"/>
+            </when>
+        </conditional>  
+
+        <param name="unstranded" type="boolean" checked="False" truevalue="-x"
+            falsevalue="" label="Don't preserve strand (puts all the peaks in positive strand)"/>
```
Comment (TorHou) [inline tools/piranha/piranha.xml]: same as above .. 
label="Preserve strand..." checked="True" ....

**Comment by bgruening** [comment]
Thanks @TorHou for the review, I tried to fix all of the issues.

**Pull request merged**